pep-0001.txt

```diff
@@ -659,6 +660,9 @@ archives, rather than just a mailto: or hyperlink to the list itself.
 The Type header specifies the type of PEP: Standards Track,
 Informational, or Process.
 
+The optional Topic header specifies which topic, if any, the PEP belongs in.
+See https://peps.python.org/topic/ for the existing topics.
```

Comment:
I don't see `Governance` at this URL.  I might also suggest `peps.python.org/topics` redirect to `/topic` (or vice versa).

Reply:
Yeah, I make this mistake all the time. This can be added in the Fastly config here: https://github.com/python/peps/blob/main/infra/main.tf

Reply:
Like this?

```tf
  snippet {
    name    = "topics"
    type    = "recv"
    content = <<-EOT
        if (req.url == "/topics") {
          set req.url = "/topic/";
        }
    EOT
  }
```

Do we also need something for "topics/" ?

Reply:
I was about to mention that. If my understanding of the syntax is correct from looking at some of the others, you can do something like ``req.url ~ "^/topics/?$"`` instead to handle that.